src/main/kotlin/no/nav/syfo/client/narmesteleder/NarmesteLederClient.kt

```diff
@@ -56,6 +58,54 @@ class NarmesteLederClient(
         }
     }
 
+    suspend fun getAktiveAnsatte(narmesteLederIdent: PersonIdentNumber, callId: String): List<NarmesteLederDTO> {
+        val systemToken = azureAdV2Client.getSystemToken(
+            scopeClientId = narmestelederClientId,
+        )?.accessToken
+            ?: throw RuntimeException("Failed to request access to current Narmesteleder: Failed to get System token")
```

Comment:
Metoden heter `getAktiveAnsatte`, men i feilmeldingen så står det `Failed to request access to current Narmesteleder`. Det trenger å fikses.